shopperz/api/views.py

```diff
@@ -3,11 +3,19 @@
 from .models import User
 from .serializers import UserSerializer
 from rest_framework.views import APIView
+from rest_framework import status
+
 
 # Create your views here.
 
-class AllViews(APIView):
-    def get(self, request):
-        user = User.objects.all()
-        serializer = UserSerializer(user, many=True)
-        return JsonResponse(serializer.data, status=200, safe=False)
+
+class Register(APIView):
```

Comment:
it would be nice to use comments to describe what the class methods do, the method args and what they return.
```
"""
		This methods creates a new job category instance
		args:
		request(dictionary)-the data attribute of the request should contain:
			- category_name(string)- the name of the job category
			- description(string)- the description of the category
			- image(string)- the image to represent the category
			
		- Returns Response with a record of the job instance created 
		- Returns Status of the API call = (201 -> created), (400 -> Bad request)
	"""
```